packages/sku/src/entry/server/server.ts

```diff
@@ -14,7 +15,7 @@ const serverOptions = serverExports({ publicPath });
 
 const { middleware, onStart, renderCallback } = serverOptions;
 
-const app = express();
+const app: Express = express();
```

Comment:
If express has types why does this need to be declared?

Reply:
It throws up a typescript error telling me to declare the type.

```
TS2742: The inferred type of app cannot be named without a reference to
`.pnpm/@types+express-serve-static-core@4.19.6/ node_modules/@types/ express-serve-static-core`
. This is likely not portable. A type annotation is necessary.
```

Which is why I typed it.